bleak/backends/bluezdbus/manager.py

```diff
@@ -166,10 +173,26 @@ def __init__(self):
 
         self._advertisement_callbacks: List[CallbackAndState] = []
         self._device_removed_callbacks: List[DeviceRemovedCallbackAndState] = []
-        self._device_watchers: Set[DeviceWatcher] = set()
-        self._condition_callbacks: Set[Callable] = set()
+        self._device_watchers: Dict[str, list[DeviceWatcher]] = {}
```

Comment:
Would set be better than list?

Also we need to support Python < 3.10, so...
```suggestion
        self._device_watchers: Dict[str, List[DeviceWatcher]] = {}
```